libethereum/Account.cpp

```diff
@@ -139,21 +144,45 @@ AccountMap dev::eth::jsonToAccountMap(std::string const& _json, u256 const& _def
 
 			u256 nonce = haveNonce ? u256Safe(o[c_nonce].get_str()) : _defaultNonce;
 
-			if (haveCode)
-			{
-				ret[a] = Account(nonce, balance);
-				if (o[c_code].type() == json_spirit::str_type)
-				{
-					if (o[c_code].get_str().find("0x") != 0 && !o[c_code].get_str().empty())
-						cerr << "Error importing code of account " << a << "! Code needs to be hex bytecode prefixed by \"0x\".";
-					else
-						ret[a].setCode(fromHex(o[c_code].get_str()));
-				}
-				else
-					cerr << "Error importing code of account " << a << "! Code field needs to be a string";
-			}
-			else
-				ret[a] = Account(nonce, balance);
+            ret[a] = Account(nonce, balance);
+            auto codeIt = o.find(c_code);
+            if (codeIt != o.end())
+            {
+                auto& codeObj = codeIt->second;
+                if (codeObj.type() == json_spirit::str_type)
+                {
+                    auto& codeStr = codeObj.get_str();
+                    if (codeStr.find("0x") != 0 && !codeStr.empty())
```

Comment:
codeStr.find("0x")  could produce an error. 
0x should be at located at first 2 bytes of the code field.

Reply:
If you look at the diff, this code is as is before. No changes.

Reply:
I know. I just saw an issue.

Reply:
I think that could be addressed in a separate PR then?